consensus/istanbul/backend/backend_test.go

```diff
@@ -184,6 +185,35 @@ func TestGetProposer(t *testing.T) {
 	}
 }
 
+func TestIsQBFTConsensus(t *testing.T) {
+	// Uses Istanbul DefaultConfig which has qbftBlock set to 0
+	chain, engine := newLegacyBlockChain(1)
+
+	qbftConsensus := engine.IsQBFTConsensus()
+	if qbftConsensus {
+		t.Errorf("IsQBFTConsensus() should return false")
+	}
+
+	// Set the value of qbftBlock to 1
+	engine.config.QbftBlock = big.NewInt(1)
+	qbftConsensus = engine.IsQBFTConsensus()
+	if qbftConsensus {
+		t.Errorf("IsQBFTConsensus() should return false")
+	}
+
+	// Create an insert a new block into the chain.
+	block := makeBlock(chain, engine, chain.Genesis())
+	_, err := chain.InsertChain(types.Blocks{block})
+	if err != nil {
+		t.Errorf("Error inserting block: %v", err)
+	}
+
+	qbftConsensus = engine.IsQBFTConsensus()
+	if !qbftConsensus {
+		t.Errorf("IsQBFTConsensus() should return true after block insertion")
+	}
```

Comment:
cleanup/teardown required?

Reply:
Pushed a commit to stop the engine at the end of test